config/sample_manifest_excel/columns.yml

```diff
@@ -783,8 +784,8 @@ disease:
   unlocked: true
   conditional_formattings:
     empty_cell:
-donor_id2:
-  name: :donor_id
+legacy_donor_id:
```

Comment:
I renamed this to make it clearer what it is. Plus I have a horrible feeling that we've probably been inserting this heading into our manifests, rather than the correct one.

Reply:
Yes, I tested before and after your change. With the Default Plate manifest, which has the `:donor_id` column:
- Before your change, heading was DONOR ID (required for cancer samples)
- After your change, heading was DONOR ID (required for EGA)
- Both uploaded into the donor_id field on sample_metadata

So the column in several manifests will have been displaying the wrong column heading, but at least the data is in the right place.

Is it worth actually removing this `:legacy_donor_id` column, as I don't think it's used anywhere now you've removed it from the Heron manifest?

Also, I searched for `donor_id2` in the repo and there are still quite a few references hanging around in the spec folder.

Reply:
Legacy_donor_id will allow the upload of manifests using the old column headers, and ensure donor_id still gets correctly updated. I did consider adding a 'legacy_headers' section to the config instead, but this would require a few more changes, including to code that is currently very general purpose.

Reply:
I see, thanks. Could you add a comment in the yml to explain why `legacy_donor_id` is still present? I can see someone removing it in the future.

Reply:
Just occurred to me that I need to do something similar for the date columns in the compliance branch.